workflows/funcscan.nf

```diff
@@ -122,21 +122,24 @@ workflow FUNCSCAN {
     */
 
     // Some tools require annotated FASTAs
-    if ( ( params.run_arg_screening && !params.arg_skip_deeparg ) || ( params.run_amp_screening && ( !params.amp_skip_hmmsearch || !params.amp_skip_amplify || !params.amp_skip_ampir ) ) || ( params.run_bgc_screening && ( !params.bgc_skip_hmmsearch || !params.bgc_skip_antismash ) ) ) {
+    if ( ( params.run_arg_screening && !params.arg_skip_deeparg ) || ( params.run_amp_screening && ( !params.amp_skip_hmmsearch || !params.amp_skip_amplify || !params.amp_skip_ampir ) ) || ( params.run_bgc_screening ) ) {
         ANNOTATION( ch_input_for_annotation )
         ch_versions = ch_versions.mix( ANNOTATION.out.versions )
-        ch_multiqc_files = ch_multiqc_files.mix( ANNOTATION.out.multiqc_files )
+
+        ch_multiqc_files = ch_multiqc_files.mix( ANNOTATION.out.multiqc_files.collect{it[1]} )
 
         ch_new_annotation = ch_input_for_annotation
                                 .join( ANNOTATION.out.faa )
                                 .join( ANNOTATION.out.gbk )
 
     } else {
         ch_new_annotation = Channel.empty()
+        ch_multiqc_files = ch_multiqc_files.mix( Channel.of([]) )
     }
 
     // Mix back the preannotated samples with the newly annotated ones
     ch_prepped_input = ch_intermediate_input.preannotated
+                        .mix( ch_intermediate_input.fastas )
```

Comment:
This is causing the failures: macrel is getting two times files with the same name. Why have you added this?

Reply:
Ah okay!
- Why I added it: There was a Macrel error making me think the original samplesheet FASTAs (without annotation) were not passed into the AMP workflow. Now you deleted my line and it magically works?!
- Now looking at ll. 83-102 again, the original FASTAs should be included in the `.preannotated` branch. So I assume it's all right now.